services/ui-src/src/measures/2024/CPAAD/questions/DefinitionOfPopulation.tsx

```diff
@@ -14,53 +61,25 @@ export const DefinitionOfPopulation = () => {
       <CUI.Heading size="sm" as="h3">
         Definition of population included in the survey sample
       </CUI.Heading>
-      <CUI.Text mt="3">
+      <CUI.Text mt="3" mb="3">
         Please select all populations that are included. For example, if your
-        data include both non-dual Medicaid beneficiaries and Medicare and
-        Medicaid Dual Eligibles, select both:
+        survey sample includes both non-dual Medicaid (Title XIX) beneficiaries
+        and Individuals Dually Eligible for Medicare and Medicaid, select both
+        Medicaid (Title XIX) and Individuals Dually Eligible for Medicare and
+        Medicaid. 
       </CUI.Text>
-      <CUI.UnorderedList m="5" ml="10">
-        <CUI.ListItem>Survey sample includes Medicaid population</CUI.ListItem>
-        <CUI.ListItem>
-          Survey sample includes Medicare and Medicaid Dually-Eligible
-          population
-        </CUI.ListItem>
-      </CUI.UnorderedList>
+
       <QMR.Checkbox
         {...register("DefinitionOfSurveySample")}
-        options={[
-          {
-            displayValue: "Survey sample includes Medicaid population",
-            value: "SurveySampleIncMedicaidPop",
-          },
-          {
-            displayValue:
-              "Survey sample includes CHIP population (e.g. pregnant women)",
-            value: "SurveySampleIncCHIP",
-          },
-          {
-            displayValue:
-              "Survey sample includes Medicare and Medicaid Dually-Eligible population",
-            value: "SurveySampleIncMedicareMedicaidDualEligible",
-          },
-          {
-            displayValue: "Other",
-            value: "SurveySampleIncOther",
-            children: [
-              <QMR.TextInput
-                formLabelProps={{ fontWeight: "400" }}
-                label="Specify:"
-                {...register("DefinitionOfSurveySample-Other")}
-              />,
-            ],
-          },
-        ]}
-      />
-      <QMR.TextArea
-        label="If this measure has been reported by the state previously and there has been a change in the included population, please provide any available context below:"
-        formControlProps={{ paddingTop: "15px" }}
-        {...register("DefinitionOfSurveySample-Changes")}
+        options={coresetId === "ACSC" ? ACSCOptions : ACSMOptions}
```

Comment:
how do you feel about adding child chip measures check here too just so that child chip uses the chip options? 

otherwise it looks great, i love the default to medicaid since all besides chip will use medicaid options
```suggestion
        options={coresetId === "ACSC" || coresetId === "CCSC" ? ACSCOptions : ACSMOptions}
```